src/app/frontend/common/components/horizontalpodautoscalers/template.html

```diff
@@ -0,0 +1,100 @@
+<!--
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<kd-card [initialized]="initialized"
+         role="table">
+  <div title>Horizontal Pod Autoscalers</div>
+
+  <div description>
+    <div class="kd-inline-property"
+         *ngIf="horizontalpodautoscalers?.length">
+      <span class="kd-muted-light">Items:&nbsp;</span>
+      <span>{{horizontalpodautoscalers.length}}</span>
+    </div>
+  </div>
+
+  <div content>
+    <mat-table [dataSource]="getDataSource()"
+               class="kd-table-no-footer">
+      <ng-container [matColumnDef]="getHorizontalpodautoscalersColumns()[0]">
+        <mat-header-cell *matHeaderCellDef>Name</mat-header-cell>
+        <mat-cell *matCellDef="let horizontalpodautoscalers">
+          <ng-container *ngIf="horizontalpodautoscalers.objectMeta.name">{{horizontalpodautoscalers.objectMeta.name}}</ng-container>
+          <ng-container *ngIf="!horizontalpodautoscalers.objectMeta.name">-</ng-container>
+        </mat-cell>
+      </ng-container>
+
+      <ng-container [matColumnDef]="getHorizontalpodautoscalersColumns()[1]">
+        <mat-header-cell *matHeaderCellDef>Namespace</mat-header-cell>
+        <mat-cell *matCellDef="let horizontalpodautoscalers">
+          <ng-container *ngIf="horizontalpodautoscalers.objectMeta.namespace">{{horizontalpodautoscalers.objectMeta.namespace}}</ng-container>
+          <ng-container *ngIf="!horizontalpodautoscalers.objectMeta.namespace">-</ng-container>
+        </mat-cell>
+      </ng-container>
+
+      <ng-container [matColumnDef]="getHorizontalpodautoscalersColumns()[2]">
```

Comment:
I think you can drop UID. It may be shown on the details page, but I would drop it from here. I think we are not showing it on other lists too.